cmd/admin-bucket-remote-add.go

```diff
@@ -183,22 +192,40 @@ func fetchRemoteTarget(cli *cli.Context) (sourceBucket string, bktTarget *madmin
 	if !madmin.ServiceType(serviceType).IsValid() {
 		fatalIf(errInvalidArgument().Trace(serviceType), "Invalid service type. Valid option is `[replication]`.")
 	}
-
+	bandwidthStr := cli.String("bandwidth")
+	bandwidth, err := getBandwidthInBytes(bandwidthStr)
+	if err != nil {
+		fatalIf(errInvalidArgument().Trace(bandwidthStr), "Invalid bandwidth number")
+	}
 	console.SetColor(cred, color.New(color.FgYellow, color.Italic))
 	creds := &auth.Credentials{AccessKey: accessKey, SecretKey: secretKey}
 	bktTarget = &madmin.BucketTarget{
-		TargetBucket: TargetBucket,
-		Secure:       secure,
-		Credentials:  creds,
-		Endpoint:     host,
-		Path:         path,
-		API:          "s3v4",
-		Type:         madmin.ServiceType(serviceType),
-		Region:       cli.String("region"),
+		TargetBucket:   TargetBucket,
+		Secure:         secure,
+		Credentials:    creds,
+		Endpoint:       host,
+		Path:           path,
+		API:            "s3v4",
+		Type:           madmin.ServiceType(serviceType),
+		Region:         cli.String("region"),
+		BandwidthLimit: int64(bandwidth),
 	}
 	return sourceBucket, bktTarget
 }
 
+func getBandwidthInBytes(bandwidthStr string) (bandwidth uint64, err error) {
+	if bandwidthStr != "" {
+		bandwidthStr = strings.ToLower(bandwidthStr)
+		bandwidthStr = strings.TrimSuffix(bandwidthStr, "ps")
```

Comment:
why do we need to support this ? is it like Gbps ?

Reply:
Yes